runner/src/mill/runner/MillBuildRootModule.scala

```diff
@@ -208,9 +208,7 @@ object MillBuildRootModule {
         s"_root_.mill.main.RootModule.Foreign(Some(_root_.mill.define.Segments.labels($segsList)))"
       }
 
-    s"""
-       |package ${pkg.mkString(".")}
-       |import _root_.mill._
+    s"""package ${pkg.mkString(".")}
        |import mill.runner.MillBuildRootModule
```

Comment:
While we're here, we should fix the hygiene and add a `_root_` to the of `MillBuildRootModule`.